src/ndn/client_conf.py

```diff
@@ -108,13 +108,15 @@ def default_face(face: str) -> Face:
     scheme, uri = face.split('://')
     if scheme == 'unix':
         return UnixFace(uri)
-    elif scheme == 'tcp' or scheme == 'tcp4':
-        if uri.find(':') >= 0:
-            host, port = uri.split(':')
-            port = port
-        else:
-            host = uri
-            port = 6363
+    if uri.find(':') >= 0:
```

Comment:
This will not work correctly with literal IPv6 address.
Use `urllib.parse`, https://stackoverflow.com/a/53172593